.github/workflows/ci-publish.yml

```diff
@@ -54,13 +55,15 @@ jobs:
       - name: Install from build
         run: |
           python -m pip install --upgrade pip
-          pip install resqpy[tests] --find-links=dist/
+          pip install poetry
+          pip install poetry-dynamic-versioning
+          poetry install
 
       - name: Checkout code
         uses: actions/checkout@v2
 
       - name: Run tests
-        run: pytest
+        run: poetry run pytest
```

Comment:
Continuing the other thread about tests, here is perhaps where we could use a `--requirebuild` flag to enable extra tests, specifically the one that checks the version number is nonzero.